### ISSUE TITLE
Use database to persistently store tweets.

### ISSUE THREAD

**Issue by cezarsa** [cezarsa/silver_bird#17]
Use html 5 database API to store tweets. This should reduce the number of calls to Twitter's API and improve our performance.

**Comment by xavivars** [comment]
It also would be useful to show unread tweets between sessions.

**Comment by cezarsa** [comment]
Yeah, I still have to figure out some safe and lightweight way of doing this, as potentially there could be thousands of unread tweets between sessions, but that's an issue I'll surely try to address soon.

**Comment by jcc8** [comment]
Bounty here: https://www.catincan.com/bounty/https-github-com-cezarsa-silverbird-issues-17

Crowdfund issue & get merged into main branch to collect.
